ming_sim/audience_night.py

```
@@ -938,34 +939,24 @@ def wait_in_flight_clear(
     *,
     timeout_s: float | None = None,
     poll_s: float | None = None,
+    write_gate: Any = None,
 ) -> None:
-    """等在飞回话完成；超时 fail-closed（夜保持开）。不持 write_gate。"""
-    if timeout_s is None:
-        timeout_s = DEFAULT_IN_FLIGHT_WAIT_S
+    """等在飞回话完成；只依 chat turn/worker 终态放行，不按 elapsed 伪造失败。
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF002)